packages/selfcare-client-users-updater/src/services/selfcareClientUsersUpdaterProcessor.ts

```diff
@@ -108,12 +108,34 @@ export function selfcareClientUsersUpdaterProcessorBuilder(
           (client) => client.adminId === jsonPayload.user.userId
         );
 
-        // TODO: For each client, call Auth Process to remove the userId from that clientId.
-
         loggerInstance.info(
           `Found ${clients.length} clients with user as admin`
         );
 
+        await Promise.all(
+          clients.map(async (client) => {
+            try {
+              await authorizationProcessClient.client.internalRemoveClientAdmin(
```

Comment:
```suggestion
             authorizationProcessClient.client.internalRemoveClientAdmin(
```
In order to make use of the `Promise.all` we shouldn't await its promises bu only the `Promise.all`